packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -23,49 +23,88 @@ import './page.css';
 
 ## Examples
 
-### Vertical nav
+### Basic page components
+
+A page will typically contain the following components: 
+
+- A `<Page>` with a `header` that often contains a masthead, a `<PageSidebar>`, and one or more `<PageSection>` components. 
+-[Mastheads](/components/masthead) contain the `<PageToggleButton>`, a `<MastheadMain>` that contains a `<MastheadBrand>`, and the page's header toolbar within `<MastheadContent>`. 
+
+### Vertical navigation
+
+To add vertical side navigation to a `<Page>`, pass a `<PageSidebar>` component into the `sidebar` property. To render navigation in the `<PageSidebar>`, use the `nav` property. 
+
+The `isNavOpen` property helps facilitate the opening and closing of the sidebar and should be 'true' when the navigation sidebar is visible. 
 
 ```ts file="./PageVerticalNav.tsx"
 ```
 
-### Horizontal nav
+### Legacy vertical navigation
+
+This example shows the legacy implementation of a page's vertical navigation. Our updated recommendation advises you to use a masthead and toolbar to make headers, rather than `<PageHeader>` and `<PageHeaderTools>` as shown in the following example.
+
+```ts file="./PageVerticalNavUsingPageHeaderComponent.tsx"
+```
+
+### Horizontal navigation
+
+To add horizontal top navigation to a `<Page>`, each navigation item should be set as a `<ToolbarItem>` in the `<Toolbar>` that is passed to the `<Masthead>`.
 
 ```ts file="./PageHorizontalNav.tsx"
 ```
 
-### Tertiary nav
+### Tertiary navigation
+
+To add a tertiary navigation component to a page, use the `tertiaryNav` property. The following example passes a string of ‘Navigation’ to `tertiaryNav`, but you can pass fully built-out components, such as navigation.
```

Comment:
Passing navigation is what this is for - the reason this is a unique page section is because it provides spacing that works with the tertiary nav in context of other stuff - breadcrumbs, tabs, other page sections, the tertiary nav horizontal scroll buttons...

Technically you could put something else in here if you wanted, but that's probably not advised in most instances. I think we should communicate that it's for tertiary nav. Inline with my comment above, we have a unique page section for breadcrumbs, tertiary nav, horizontal nav, tabs... they're ideally designed for those respective components.